### DISCUSSION TITLE
Quick Support Client version

### DISCUSSION THREAD

**Discussion by alkinabc** [rustdesk/rustdesk#8981] [Q&A] [answered]
Good day,
I would use RustDesk in the company I work for. For this we would need a quick support solution on the RustDesk client. What I mean by that is that, like with Teamviewer, there should be a client that only receives and does not send.

**Comment by rustdesk** [comment, marked as answer]
With RustDesk server pro (basic plan or custom plan), you can build custom client, there is portable quick support version (built-in server ip and key, also a lot of options configurable).

https://rustdesk.com/pricing/?lang=en#:~:text=Custom%20client%20generator

https://x.com/rustdesk/status/1769171628426944539

https://www.youtube.com/watch?v=FKXyUClA6pY

# What about casual support cases? We have some devices that need to be supported occasionally without unattended access.

We provide this [solution](https://github.com/rustdesk/rustdesk-server-pro/discussions/182#discussioncomment-10083053)

**Reply by rustdesk** [reply]
![image](https://github.com/user-attachments/assets/9d599313-57cd-4035-a1ab-254e25ba7a27)

With `Incoming` on, you will get something like this. You might also turn  on `Disable installation`.

![image](https://github.com/user-attachments/assets/ad899930-e067-4214-811a-2332c2125d8c)

**Reply by rustdesk** [reply]
# Here is step by step.

- Go to custom client page of web console.

![image](https://github.com/user-attachments/assets/db073d9e-047d-4487-8f34-f5b3c2bdb107)

- Click on `Create`

![image](https://github.com/user-attachments/assets/8a269774-0e20-4e2e-a5ab-4239d96dfebf)

- Turn on `Incoming` option.

![image](https://github.com/user-attachments/assets/9d599313-57cd-4035-a1ab-254e25ba7a27)

- Click on `Create` after you finish

![image](https://github.com/user-attachments/assets/098ce040-43f8-4802-aec8-537af7fcb050)

- Click on below `Build` action

![image](https://github.com/user-attachments/assets/6637f9af-a32d-4a31-84e8-62162acff44e)

- After one moment, you will see below `Download` link. It might take several minutes to half hour, depending on if the build server is busy

![image](https://github.com/user-attachments/assets/5acb3143-ed51-49d4-9b35-c568e119427a)

**Comment by rustdesk** [comment]
![rustdeskcreateclient](https://github.com/user-attachments/assets/337fe7eb-4ce0-4841-95b4-d32172d2f3e2)
